app/views/training/modules/_subsection.html.slim

```diff
@@ -1,6 +1,8 @@
 span.module-section--item
-  - if next_page_name
-    = govuk_link_to heading, training_module_page_path(mod_name, next_page_name)
-  - else
-    = heading
+  ul
+    li
+      - if next_page_name
+        = govuk_link_to heading, training_module_page_path(mod_name, next_page_name)
+      - else
+        = heading
```

Comment:
Is this a list of one?

Reply:
The ticket says that it should be implemented this way (each bullet point has its own list)